pywb/static/wombat.js

```diff
@@ -3630,6 +3804,9 @@ var _WBWombat = function($wbwindow, wbinfo) {
             }
 
             send_top_message(message);
+            if (WBPreserWorker) {
```

Comment:
check `wbUsePresWorker` for consistency?

Reply:
`notify_top()` actually runs twice (for readyState `interactive` and for readyState `complete`, to allow the banner to update as soon as possible). Probably should send to worker only when readyState is `complete`?